test/train-sets/ref/help.stdout

```diff
@@ -539,6 +539,12 @@ Weight Options:
                                             uint, keep, necessary)
     --replay_m_count arg                    How many times (in expectation) should each example be played
                                             (default: 1 = permuting) (type: uint, default: 1)
+[Reduction] Experimental: Contextual Bandit Exploration with ADF with large action space Options:
+    --cb_explore_adf                        Online explore-exploit for a contextual bandit problem with multiline
+                                            action dependent features (type: bool, keep, necessary)
+    --large_action_space                    Large action space exploration (type: bool, keep, necessary)
+    --max_actions arg                       Max number of actions to explore (type: uint, default: 50, keep)
+    --gamma arg                             Gamma hyperparameter (type: float, keep)
```

Comment:
the `--gamma` hyperparameter is from squarecb.  we should reuse the arguments from there (there's another one iirc).

Reply:
synced offline, will get this reduction to call squarecb reduction